### PR TITLE
Add initial about page

### PR DESCRIPTION
**Pull request by domdomegg**

### CODE REVIEW

File: web/src/pages/about.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      <p>There's {NATIONAL_TEAM.length} of us on the national team, working to support chapters as best we can:</p>
+      {NATIONAL_TEAM.map((m, i) => (
+        <Quote key={m.id} by={`${m.name}, ${m.role}`} headshotSrc={m.img} imagePlacement={i % 4 <= 1 ? "left" : "right"} className="mt-4 p-8 bg-raise-red rounded-lg min-h-[10rem] whitespace-pre-line">
+          {m.bio}
+        </Quote>
+      ))}
+      <p className="mt-4">You can contact the national team via email at raisenational@gmail.com.</p>
+
+      <p className="font-black font-raise-header mt-8 mb-2">Advisory Board</p>
+      <p>The advisory board is group of Raise alumni, most of whom have previously served on chapter teams.</p>
```
Comment (domdomegg) [inline web/src/pages/about.tsx:116]: This will be expanded upon in future

File: web/src/pages/about.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        { text: "Alumni", href: "/alumni/" },
+        { text: "Bristol", href: "/bristol/" },
+        { text: "Cambridge", href: "https://www.mayweekalternative.org.uk/" },
+        { text: "Durham", href: "/durham/" },
+        { text: "Edinburgh", href: "/edinburgh/" },
+        { text: "Glasgow", href: "/glasgow/" },
+        { text: "Oxford", href: "/oxford/" },
+        { text: "Sheffield", href: "/sheffield/" },
+        { text: "Warwick", href: "/warwick/" },
+        { text: "About", href: "/about/" },
```
Comment (domdomegg) [inline web/src/pages/about.tsx:87]: Will edit the navigation on the other pages once we're happy internally with making this 'more' public.

**Comment by domdomegg** [comment]
NB: photos updated will also reflect on homepage

**Comment by domdomegg** [comment]
image
image
image
image

**Review by Malena205 — APPROVED** [review]

**Pull request merged**